Source/CBLView+REST.m

```diff
@@ -16,16 +16,18 @@ @implementation CBLView (REST)
 
 
 - (CBLStatus) compileFromDesignDoc {
-    if (self.registeredMapBlock != nil)
-        return kCBLStatusOK;
-
     // see if there's a design doc with a CouchDB-style view definition we can compile:
     NSString* language;
     NSDictionary* viewProps = $castIf(NSDictionary, [self.database getDesignDocFunction: self.name
                                                                                     key: @"views"
                                                                                language: &language]);
-    if (!viewProps)
-        return kCBLStatusNotFound;
+    if (!viewProps) {
+        if (self.registeredMapBlock != nil)
+            return kCBLStatusOK;
```

Comment:
This change means that after the design doc is deleted, or the map function is removed from the design doc, the view continues to work. Isn't that wrong?